src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+            },
+        ];
+    } else if (assembly === 'ce11') {
+        // Genome: from ftp://hgdownload-sd.sdsc.edu/goldenPath/ce11/bigZips/ce11.2bit
+        browserCfg.chr = 'II';
+        browserCfg.viewStart = 14646376;
+        browserCfg.viewEnd = 14667875;
+        browserCfg.cookieKey = 'WBcel235';
+        browserCfg.coordSystem = { speciesName: 'Caenorhabditis elegans', taxon: 6239, auth: 'WBcel', version: 235 };
+        browserCfg.sources = [
+            {
+                name: 'Genome',
+                twoBitURI: 'https://s3-us-west-1.amazonaws.com/encoded-build/browser/ce11/ce11.2bit',
+                desc: 'C. elegans reference genome build WBcel235',
+                tier_type: 'sequence',
+                provides_entrypoints: true,
+            },
+        ];
+    }
+
+    return browserCfg;
+}
+
+
+const GenomeBrowser = React.createClass({
+    propTypes: {
+        files: React.PropTypes.array.isRequired, // Array of files to represent
+        assembly: React.PropTypes.string.isRequired, // Assembly to use with browser
+        region: React.PropTypes.string, // Region to use with browser
+        visBlobs: React.PropTypes.object, // This should contain one or more vis_blobs for dataset(s)
+        limitFiles: React.PropTypes.bool, // True to limit # files to maxFilesBrowsed
+    },
+
+    contextTypes: {
+        location_href: React.PropTypes.string,
+        localInstance: React.PropTypes.bool,
+    },
+
+    componentDidMount: function () {
+        const { assembly, region, limitFiles } = this.props;
+        console.log('ASSEMBLY: %s', assembly);
+
+         // Probably not worth a define in globals.js for visualizable types and statuses.
+        // Extract only bigWig and bigBed files from the list:
+        let files = this.props.files.filter(file => file.file_format === 'bigWig' || file.file_format === 'bigBed');
+        files = files.filter(file =>
+                    ['released', 'in progress', 'archived'].indexOf(file.status) > -1
+                );
+
+
+        // Make some fake file objects from "test" just to give the genome browser something to
+        // chew on if we're running locally.
+        files = !this.context.localInstance ?
+            (limitFiles ? files.slice(0, maxFilesBrowsed - 1) : files)
+        : dummyFiles;
+
+        const browserCfg = rAssemblyToSources(assembly, this.props.region);
+
+        this.browserFiles = [];
+        let domain = `${location.protocol}//${location.hostname}`;
+        if (domain.includes('localhost')) {
+            domain = domainName;
+        }
+        files.forEach((file) => {
+            const trackLabels = this.makeTrackLabel(file);
+            if (file.file_format === 'bigWig') {
+                this.browserFiles.push({
+                    name: trackLabels.shortLabel,
+                    desc: trackLabels.longLabel,
+                    bwgURI: `${domain}${file.href}`,
+                    style: [
```

Comment:
Without giving it a _whole_ lot of thought, that does sound like a good idea. Not sure how visualize.py handles it.